eng/common/pipelines/templates/jobs/sdk-generation.yml

```diff
@@ -38,6 +27,8 @@ jobs:
       value: '22.13.x'
     - name: PythonVersion
       value: '3.13'
+    - name: SpecGenSdkVersion
+      value: 'latest'
```

Comment:
I'm ok with latest for now but we will eventually want to pin this to the last known good version instead of floating.